modules/medianetBidAdapter.js

```diff
@@ -132,27 +132,20 @@ function getCoordinates(id) {
   return null;
 }
 
-function extParams(params, gdpr, uspConsent) {
-  let ext = {
-    customer_id: params.cid,
-    prebid_version: $$PREBID_GLOBAL$$.version
-  };
-  ext.gdpr_applies = !!(gdpr && gdpr.gdprApplies);
-  if (ext.gdpr_applies) {
-    ext.gdpr_consent_string = gdpr.consentString || '';
-  }
-
-  ext.usp_applies = !!(uspConsent);
-  if (ext.usp_applies) {
-    ext.usp_consent_string = uspConsent || '';
-  }
-
+function extParams(params, gdpr, uspConsent, userId) {
   let windowSize = spec.getWindowSize();
-  if (windowSize.w !== -1 && windowSize.h !== -1) {
-    ext.screen = windowSize;
-  }
-
-  return ext;
+  let gdprApplies = !!(gdpr && gdpr.gdprApplies);
+  let uspApplies = !!(uspConsent);
+  return Object.assign({},
```

Comment:
I think technically this first param {} is not needed since the next ones will always return something.

But no big deal.